src/Mod/PartDesign/Gui/TaskChamferParameters.cpp

```diff
@@ -96,6 +97,9 @@ TaskChamferParameters::TaskChamferParameters(ViewProviderDressUp *DressUpView, Q
     createDeleteAction(ui->listWidgetReferences, ui->buttonRefRemove);
     connect(deleteAction, SIGNAL(triggered()), this, SLOT(onRefDeleted()));
 
+    createSelectAllEdgesAction(ui->listWidgetReferences);
+    connect(selectAllEdgesAction, SIGNAL(triggered()), this, SLOT(onSelectAllEdges()));

```

Comment:
In new code the overloaded version of connect() that uses the SIGNAL and SLOT macros should be avoided. Instead use the version where you function pointers are used.
`
connect(selectAllEdgesAction, &QAction::triggered, this, &TaskChamferParameters::onSelectAllEdges);
`
The advantage is that in case a signal or slot function isn't defined the compiler already raises an error.